src/transforms/uglify.js

```diff
@@ -5,7 +5,7 @@ const walk = require('babylon-walk');
 module.exports = async function(asset) {
   await asset.parseIfNeeded();
 
-  // Convert to UglifyES AST
+  // Minify with UglifyES
```

Comment:
Suggest to change to "Minify with uglify-es" to match the npm package name. The term "UglifyES" is not used by its maintainers.

Reply:
Will do, tnx for the suggestion 💯

Reply:
...or better yet - drop the comment altogether.